yt/frontends/ytdata/data_structures.py

```diff
@@ -576,9 +576,6 @@ def __init__(self, gid, index, filename=None):
         self.LeftEdge = self.index.ds.domain_left_edge
         self.RightEdge = self.index.ds.domain_right_edge
 
-    def __repr__(self):
```

Comment:
I don't think that this has `ActiveDimensions`, right?  So removing this override may introduce an error.

Reply:
This line implies (maybe erroneously) that all instances of `AMRGridPatch` have an `ActiveDimensions` attribute https://github.com/yt-project/yt/blob/0fbc997f77bf18c0b45419d4cca06a294fb09516/yt/data_objects/index_subobjects/grid_patch.py#L84